### PR TITLE
week 1 assignment

### PR DESCRIPTION
**Pull request by diegofalconcostilla**
* Added background color functionality to a button
* Added api connection and functionality to retrieve a random word

### CODE REVIEW

File: button.js [hunk 1]
```diff
@@ -2,26 +2,24 @@ function changeColor() {
...
 
-  fetch("https://wordsapiv1.p.rapidapi.com/words/?random=true", {
-    method: "GET",
+  fetch('https://wordsapiv1.p.rapidapi.com/words/?random=true', {
+    method: 'GET',
     headers: {
-      "x-rapidapi-key": "your-api-key-here",
-      "x-rapidapi-host": "api-host-here",
+      'x-rapidapi-host': config["x-rapidapi-host"],
+      'x-rapidapi-key': config["x-rapidapi-key"]
```
Comment (panzerama) [inline button.js:15]: Where is config defined? This is a great approach to handling keys, I'm just curious as to where it's set up.

File: button.js [hunk 2]
```diff
@@ -2,26 +2,24 @@ function changeColor() {
   var blueDiv = document.getElementById("changeable");
   // Use the Mozilla Developer Network resource to find the right way to change
```
Comment (panzerama) [inline button.js:3]: You can remove comments from your final submission unless they serve a specific purpose.

File: button.js [hunk 3]
```diff
@@ -2,26 +2,24 @@ function changeColor() {
...
   })
     .then((response) => {
       console.log(response);
-      response.json().then(() => {
-        /* 
-          Complete this handler function with code that populates an html 
-          element with the random word 
-        */
+      response.json().then((responseBody) => {
+        console.log(responseBody.word);
```
Comment (panzerama) [inline button.js:21]: Unless a console.log statement serves a specific purpose it's best to remove it before submitting a pull request.

File: index.html [hunk 4]
```diff
@@ -5,6 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="main.css">
   <script src="button.js"></script>
+  <script type="text/javascript" src="config.js"></script>
```
Comment (panzerama) [inline index.html:8]: Very cool.

File: button.js (hunk 2 above)
Comment (diegofalconcostilla) [reply button.js:3]: Got it, i'll clean it up. Should I submit a new pull request?

File: main.css [hunk 5]
```diff
@@ -23,6 +23,9 @@
 }
 
 /* Add a different color class here */
```
Comment (panzerama) [inline main.css:25]: In the future, please make sure to review and remove unnecessary comments

**Pull request closed without merging**